benchmarks/lwbench.cpp

```diff
@@ -34,7 +34,9 @@ double A()
     }
     auto t2 = std::chrono::high_resolution_clock::now();
 
-    return std::chrono::duration_cast<std::chrono::nanoseconds>(t2 - t1).count()
+    return static_cast<double>(
+               std::chrono::duration_cast<std::chrono::nanoseconds>(t2 - t1)
+                   .count())
```

Comment:
@isuruf -Wconversion catches the implicit type conversion happening here (since the above expression returns an unsigned int. Hence the change.